pymess/models/emails.py

```diff
@@ -76,15 +76,10 @@ class EmailMessage(BaseMessage):
         blank=True,
         editable=False,
     )
-    old_content = models.TextField(
-        verbose_name=_('content'),
-        null=True,
-        blank=True
-    )
     content_file = models.FileField(
         verbose_name=_('content file'),
-        null=True,
-        blank=True,
+        null=False,
+        blank=False,
```

Comment:
but if content file is not required the content property shoud return none if file was not stored. Now you must change the content property. Otherwise exception will be raised

Reply:
But I am making it mandatory - there should not exist e-mail message without content file at this moment (new e-mails are already saved into content files and old ones were fixed by migration).